pom.xml

```diff
@@ -9,7 +9,7 @@
 
     <groupId>com.sonyericsson.hudson.plugins.gerrit</groupId>
     <artifactId>gerrit-trigger</artifactId>
-    <version>2.29.1-SNAPSHOT</version>
+    <version>2.30.0-uninterested</version>
```

Comment:
?

Reply:
Sorry, I didn't mean to commit that to the PR. I just wanted an identifier to know we were internally running something different.

I'll remove when I get into work.